components/ProfilePage/ProfileHeader.tsx

```diff
@@ -1,28 +1,113 @@
-import { Header, ProfileDisplayName } from "./StyledProfileComponents"
-import { ProfileIcon } from "./StyledUserIcons"
-
 import { useTranslation } from "next-i18next"
+import { useContext, useState } from "react"
+import { useMediaQuery } from "usehooks-ts"
+import { useAuth } from "../auth"
+import { Container, Row, Spinner } from "../bootstrap"
 import { Profile } from "../db"
-import { OrgContactInfo } from "./OrgContactInfo"
-import { ProfileButtonsOrg, ProfileButtonsUser } from "./ProfileButtons"
+import {
+  ProfileHook,
+  UseDraftTestimonyListing,
+  UsePublishedTestimonyListing,
+  useDraftTestimonyListing,
+  useProfile,
+  usePublishedTestimonyListing
+} from "../db"
+import { EditProfileButton, ProfileButtons } from "./ProfileButtons"
+import { Header, ProfileDisplayName } from "./StyledProfileComponents"
+import { ProfileIcon } from "./StyledUserIcons"
+import ProfileSettingsModal from "components/EditProfilePage/ProfileSettingsModal"
+import { FollowUserButton } from "components/shared/FollowButton"
 
-export const ProfileHeader = ({
+export function HeaderWrapper({
   profileId,
-  profile,
   isUser,
   isOrg,
-  isProfilePublic,
-  onProfilePublicityChanged
+  onProfilePublicityChanged,
+  profile
 }: {
-  profileId: string
+  isUser: boolean
+  isOrg: boolean
   profile: Profile
+  profileId: string
+  onProfilePublicityChanged: (isPublic: boolean) => void
+}) {
+  const { user } = useAuth()
+  const uid = user?.uid
+  const result = useProfile()
+
+  console.log("target: ", profile)
```

Comment:
needed a way to grab two profiles at the same time (self and target)

there's probably a more elegant way to do this